ServerCore/Pages/EventSpecific/Index.cshtml

```diff
@@ -1,7 +1,23 @@
-﻿@page "/{eventId}"
+﻿@page "/{eventId}/{eventRole}"
 @model ServerCore.Pages.EventSpecific.IndexModel
 @{
     ViewData["Title"] = Model.Event.Name;
+    ViewData["AdminRoute"] = "/EventSpecific/Index";

```

Comment:
Maybe the absence of the data means go to the index? You'd have fewer records that way.

Reply:
As in remove the word "Index" or as in remove this line entirely since it is the default? I included the line so that I wouldn't think I had forgotten to add an admin route and to keep this consistent in case the default changes, but I could clean up the word "Index"

Reply:
Will do in separate cleanup if needed